orca/src/llm/openai.rs

```diff
@@ -363,6 +417,35 @@ mod test {
         let mut context = HashMap::new();
         context.insert("country1", "France");
         context.insert("country2", "Germany");
+        let prompt = template!(
+            "my template",
+            r#"
+            {{#chat}}
+            {{#user}}
+            What is the capital of {{country1}}?
+            {{/user}}
+            {{#assistant}} 
+            Paris
+            {{/assistant}}
+            {{#user}}
+            What is the capital of {{country2}}?
+            {{/user}}
+            {{/chat}}
+            "#
+        );
+        let prompt = prompt.render_context("my template", &context).unwrap();
+        let response = client.generate(prompt).await.unwrap();
+        assert!(response.to_string().to_lowercase().contains("berlin"));
+    }
+
+    #[tokio::test]
+    async fn test_generate_json_mode() {
+        let client = OpenAI::new().with_model("gpt-3.5-turbo-1106").with_response_format(Some(ResponseFormat {
+            type_: "json".to_string(),
```

Comment:
I read "json_object" in Open AI documentation. Am I missing something?

Reply:
You are correct! Good pick up!